sylvia-derive/src/parser.rs

```diff
@@ -370,7 +372,11 @@ impl<'a> Custom<'a> {
     }
 
     pub fn msg(&self) -> &Path {
-        &self.msg
+        self.msg.as_ref().unwrap_or(&self.default_type)
```

Comment:
Ok, I get it - you wanted to return a reference here, so you needed to keep it somewhere. Makes a little sense, but also seems a bit strange. This seems like an exact case for `https://docs.rs/lazy_static/latest/lazy_static/`.

Reply:
`lazy_static` requires type to implement `Sync` which `Path` does not. I think none of the types in `syn` does.

Reply:
Yea, ok, I see this. I don't like this in general, but for now we can leave it here.